Receiver/receiverTest.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import simpleMovingAverage
+class ReceiverTest(unittest.TestCase):
+     
+    def testMinTemp(self):
+        self.assertTrue(simpleMovingAverage.calculateMin({'temperature':[20,19,1,15,18,90,60,80,70,50]},'temperature') == 1)
+        
+    def testMaxTemp(self):
+        self.assertTrue(simpleMovingAverage.calculateMax({'temperature':[20,19,10,15,18,96,60,80,70,50]},'temperature') == 96)  
+        
+    def testMinSOC(self):
+        self.assertTrue(simpleMovingAverage.calculateMin({'soc':[70,40,25,9,90,98,88,77,66,55]},'soc') == 9)
+        self.assertTrue(simpleMovingAverage.calculateMin({'temperature':[20,19,10,15,18,96,60,80,70,50],'soc':[70,40,25,9,90,98,88,77,66,55]},'temperature')==10)  
```

Comment:
Name your tests with the input and expected output. Things are not as obvious as they seem:
- Input has to contain valid numbers (no strings, not NAN, etc)
- Output would be a floating-point within a tolerance of 0.001? Because floating-point cannot be checked for exact equality. It may work this time for the input given, but may not work if the average contains recurring decimals.